### DISCUSSION TITLE
Default XS break point has higher order than other break points !

### DISCUSSION THREAD

**Discussion by pooria-noruzi** [tailwindlabs/tailwindcss#11622] [Help]
**What version of Tailwind CSS are you using?**

    "tailwindcss": "^3.2.7",

**What build tool (or framework if it abstracts the build tool) are you using?**

 "vite": "^4.1.0",

default XS break point has higher order than others , for example when i use class like this : 'xs:w-1/2 sm:w-1/3 md:w-1/4' w-1/2 come for all break points then we must use it with important label like this  'xs:w-1/2 sm:!w-1/3 md:!w-1/4 '

**Comment by adamwathan** [comment]
Hey! Will need a reproduction for this one — Tailwind has no `xs` breakpoint by default but when you add one it is automatically sorted into the right place:

https://play.tailwindcss.com/tO3VfQKKRo

Make sure you're on the latest version of Tailwind too 👍

**Reply by shrutibalasawebdev** [reply]
So we don't need to do this anymore as specified in the docs - https://tailwindcss.com/docs/screens#adding-smaller-breakpoints?

```
const defaultTheme = require('tailwindcss/defaultTheme')

module.exports = {
  theme: {
    screens: {
      'xs': '475px',
      ...defaultTheme.screens,
    },
  },
  plugins: [],
}
```

**Reply by thecrypticace** [reply]
No, it's not necessary any more. That documentation needs to be updated. I've added an internal task for us to update it.
